### PR TITLE
Release Notes 4.1.0

### PR DESCRIPTION
**Pull request by Kaur-Parminder**
Release Notes for 4.1.0

-  Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. #1419 , #1425

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+## [Stable release 4.1.0] - 2022-01-28
+
+### Added
+
+- Added New Attestation Protocol `None` for enclave type `VBS` with optional `AttestationUrl`. This connection string property when set will allow users to forgo enclave attestation for VBS enclaves. [#1425](https://github.com/dotnet/SqlClient/pull/1425)
```
Comment (johnnypham) [inline CHANGELOG.md:10]:
```suggestion
- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1425](https://github.com/dotnet/SqlClient/pull/1425)
```

File: release-notes/4.1/4.1.0.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 4.1.0 released 28 January 2022
+
+This update brings the below changes over the previous preview release:
+
+### Added
+
+- Added New Attestation Protocol `None` for enclave type `VBS` with optional `AttestationUrl`. This connection string property when set will allow users to forgo enclave attestation for VBS enclaves. [#1425](https://github.com/dotnet/SqlClient/pull/1425)
```
Comment (johnnypham) [inline release-notes/4.1/4.1.0.md:9]:
```suggestion
- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1425](https://github.com/dotnet/SqlClient/pull/1425)
```

**Review by DavoudEshtehari — COMMENTED** [review]
Update [README.md](https://github.com/dotnet/SqlClient/blob/main/release-notes/README.md) too.

**Comment by Kaur-Parminder** [comment]
> Update [README.md](https://github.com/dotnet/SqlClient/blob/main/release-notes/README.md) too.

Done

File: release-notes/4.1/4.1.0.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+### Added
+
+- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1419](https://github.com/dotnet/SqlClient/pull/1419) [#1425](https://github.com/dotnet/SqlClient/pull/1425) [Read more](#Use-Attestation-Protocol-=-None-to-skip-enclave-attestation-for-VBS-enclaves)
+
+### Use `Attestation Protocol = None` to skip enclave attestation for `VBS` enclaves.
+
+A new attestation protocol called `None` will be allowed in the connection string. This protocol will allow users to forgo enclave attestation for `VBS` enclaves. When this protocol is set, the enclave attestation URL property is optional.  
+
+Connection string examples:
```
Comment (johnnypham) [inline release-notes/4.1/4.1.0.md:15]:
```suggestion
Connection string example:
```

**Review by johnnypham — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,12 @@
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+## [Stable release 4.1.0] - 2022-01-28
```
Comment (Kaur-Parminder) [inline CHANGELOG.md:6]:
```suggestion
## [Stable release 4.1.0] - 2022-01-31
```

File: release-notes/4.1/4.1.0.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 4.1.0 released 28 January 2022
+
+This update brings the below changes over the previous preview release:
+
+### Added
+
+- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1419](https://github.com/dotnet/SqlClient/pull/1419) [#1425](https://github.com/dotnet/SqlClient/pull/1425) [Read more](#Use-Attestation-Protocol-=-None-to-skip-enclave-attestation-for-VBS-enclaves)
```
Comment (Kaur-Parminder) [inline release-notes/4.1/4.1.0.md:9]:
```suggestion
- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1419](https://github.com/dotnet/SqlClient/pull/1419) [#1425](https://github.com/dotnet/SqlClient/pull/1425) [Read more](#use-attestation-protocol-none-to-skip-enclave-attestation-for-vbs-enclaves)
```

File: release-notes/4.1/4.1.0.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+### Use `Attestation Protocol = None` to skip enclave attestation for `VBS` enclaves.
```
Comment (Kaur-Parminder) [inline release-notes/4.1/4.1.0.md:11]:
```suggestion
### Use Attestation Protocol None to skip enclave attestation for VBS enclaves
```

File: release-notes/4.1/4.1.0.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 4.1.0 released 28 January 2022
+
+This update brings the below changes over the previous preview release:
+
+### Added
+
+- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1419](https://github.com/dotnet/SqlClient/pull/1419) [#1425](https://github.com/dotnet/SqlClient/pull/1425) [Read more](#use-attestation-protocol-none-to-skip-enclave-attestation-for-vbs-enclaves)
```
Comment (Kaur-Parminder) [inline release-notes/4.1/4.1.0.md:9]:
```suggestion
- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1419](https://github.com/dotnet/SqlClient/pull/1419) [#1425](https://github.com/dotnet/SqlClient/pull/1425) [Read more](#introduce-attestation-protocol-none)
```

File: release-notes/4.1/4.1.0.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+### Use Attestation Protocol None to skip enclave attestation for VBS enclaves
```
Comment (Kaur-Parminder) [inline release-notes/4.1/4.1.0.md:11]:
```suggestion
### Introduce Attestation Protocol None
```

File: release-notes/4.1/4.1.0.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 4.1.0 released 28 January 2022
```
Comment (DavoudEshtehari) [inline release-notes/4.1/4.1.0.md:3]:
```suggestion
## Microsoft.Data.SqlClient 4.1.0 released 31 January 2022
```

File: release-notes/4.1/4.1.md [hunk 10]
```diff
@@ -0,0 +1,7 @@
+# Microsoft.Data.SqlClient 4.1 Releases
+
+The following Microsoft.Data.SqlClient 4.1 stable releases have been shipped:
+
+| Release Date | Version | Notes |
+| :-- | :-- | :--: |
+| 2022/01/28 | 4.1.0 | [release notes](4.1.0.md) |
```
Comment (DavoudEshtehari) [inline release-notes/4.1/4.1.md:7]:
```suggestion
| 2022/01/31 | 4.1.0 | [release notes](4.1.0.md) |
```

File: release-notes/4.1/README.md [hunk 11]
```diff
@@ -0,0 +1,7 @@
+# Microsoft.Data.SqlClient 4.1 Releases
+
+The following Microsoft.Data.SqlClient 4.1 stable releases have been shipped:
+
+| Release Date | Version | Notes |
+| :-- | :-- | :--: |
+| 2022/01/28 | 4.1.0 | [release notes](4.1.0.md) |
```
Comment (DavoudEshtehari) [inline release-notes/4.1/README.md:7]:
```suggestion
| 2022/01/31 | 4.1.0 | [release notes](4.1.0.md) |
```

**Review by DavoudEshtehari — APPROVED** [review]

**Review by johnnypham — APPROVED** [review]

**Review by JRahnama — APPROVED** [review]

**Pull request merged**